src/main/java/org/opentripplanner/middleware/i18n/Message.java

```diff
@@ -19,7 +19,10 @@ public enum Message {
     ACCEPT_DEPENDENT_EMAIL_SUBJECT,
     ACCEPT_DEPENDENT_EMAIL_MANAGE,
     ACCEPT_DEPENDENT_ERROR,
+    DESTINATION_CHANGE_NOTIFICATION,
```

Comment:
Rename to something like `ARRIVED_NOTIFICATION` or `AT_DESTINATION_NOTIFICATION`?

Reply:
Fixed in: https://github.com/ibi-group/otp-middleware/pull/275/commits/91d66190cb611283993b8e745148641efd695c5e